src/l500/l500-depth.cpp

```diff
@@ -515,27 +515,29 @@ namespace librealsense
                 }
 
                 try {
-                    // endpoint 2 (depth)
-                    command cmdTprocGranEp2(ivcam2::TPROC_USB_GRAN_SET, 2, ep2_usb_trb);
-                    _owner->_hw_monitor->send(cmdTprocGranEp2);
+                    // Keep the USB power on while triggering multiple calls on it.
```

Comment:
Didn't we say we'd surround the whole start with invoke_powered? Not feasible?

Reply:
We can, but I preferred not to surround the synthetic_sensor::start call which is wrapped with an actions delayer for 2 reasons:
1. not to stall the USB on high power more than it need to be
2. To avoid potential risk unwanted behavior
3. I don't think it is needed

Reply:
Okay